glusterd2/bricksplanner/subvoltype_replicate.go

```diff
@@ -44,3 +46,18 @@ func (s *replicaSubvolPlanner) BrickType(idx int) string {
 func init() {
 	subvolPlanners["replicate"] = &replicaSubvolPlanner{}
 }
+
+// ArbiterBrickSizeCalculation returns the arbiter brick size for that particular volume
+func ArbiterBrickSizeCalculation(brickSize uint64, avgFileSize uint64) (uint64, error) {
```

Comment:
Do not add new function here, add calculation where following `TODO: Calculate Arbiter brick size` is present in this file.